index.html

```diff
@@ -13,27 +13,29 @@
   <header>
     <section class="top-nav">
       <div>
-        <img id="mylogo" src="/Image/logo-david-nuevo-2021.png" alt="">
+       <a href="https://www.linkedin.com/in/david-verde-3349b114b/e"> <img id="mylogo" src="/Portfolio-setup-and-mobile-version-skeleton/Image/logo-david-nuevo-2021.png" alt="logo"> </a>
       </div>
-      <label for="menu-toggle">MY FIRST WEB PAGE</label>
-      <input id="menu-toggle" type="checkbox" />
-      <label class='menu-button-container' for="menu-toggle">
-        <span class='menu-button'></span>
-      </label>
-      <ul class="menu">
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
-      </ul>
+      <nav>
+        <label for="menu-toggle">MY FIRST WEB PAGE</label>
+        <input id="menu-toggle" type="checkbox" />
+        <label class='menu-button-container' for="menu-toggle">
+          <span class='menu-button'></span>
+        </label>
+        <ul class="menu">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
     </section>
   </header>
 
   <main>
     <section id="background">
 
-      <h2 id="name">I'm David Verde! Glad to see you! </h2>
+      <h2 id="name">I'm David Verde! Glad to see <br> you! </h2>
```

Comment:
- [ ] This should be in an `H1` tag kindly work on it
- [ ] `<p id="connect">LET'S CONNECT</p>` should be a button or a link. Kindly check that